src/main/resources/templates/home.html

```diff
@@ -1,11 +1,12 @@
 ﻿<!DOCTYPE html>
-<div>
-    <p th:text="'Hello, ' + ${twitterdiv>ile.screenName} + '!'"/>
-</div>
-<div th:each="node : ${nodes}">
-    <p th:text="${node.name}"></p>
-</div>
 <div class="col-md-8">
+    <div>
+        <p th:text="'Hello, ' + ${twitterdiv>ile.screenName} + '!'"/>
```

Comment:
there is an weird issue here, the block `${twitterdiv>ile.screenName}` should be `${twitterProfile.screenName}`. When I check git logs, I see it was modified with your change with commit hash 8549c6e8a6e3b060b3fa43c0a8f9656953961d2c on 25.03.2018. Better to revert this change to prevent errors.